### DISCUSSION TITLE
Iphone status payment brick / check out transparent

### DISCUSSION THREAD

**Discussion by dvmarthur** [mercadopago/sdk-js#111] [General]
Has anyone ever seen an error on the iphone with the bricks? in my application the payment status brick does not render

**Comment by icaldana** [comment]
Hello @dvmarthur 
We apologize for the inconvenience it may have caused to your workflow. However, we need more details about the error you are seeing, in order to assess it.

This space is not intended to be a support tool, thus we must understand whether the issue was caused by the SDK-js or by a misconfiguration on your end. For the latter scenario, **we have an official channel that is super prepared to solve your case quickly.**: https://www.mercadopago.com/developers/pt/support/center/contact

I will be waiting for more details on this.
Thank you very much for your contact!

**Reply by marianolop** [reply]
@icaldana  I had uploaded more info, could you please check?

**Comment by marianolop** [comment]
Hi, I'm having the same issue, this is my code
`<!DOCTYPE html>
<html lang="es">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Estacionar</title>
</head>
<body>

    <div id="statusScreenBrick_container"></div>

    <script src="https://sdk.mercadopago.com/js/v2"></script>
    <script>

        const mp = new MercadoPago( 'not_the_real_one', {
            locale: 'es-AR',
        } );

        const bricksBuilder = mp.bricks();
        const renderStatusScreenBrick = async (bricksBuilder) => {

            const settings = {
                initialization: {
                    paymentId: '1311582185',
                },
                callbacks: {
                    onReady: () => {
                    },
                    onError: (error) => {
                    },
                },
            };
            try {
                window.statusBrickController = await bricksBuilder.create(
                    'statusScreen',
                    'statusScreenBrick_container',
                    settings
                );
            } catch (error) {
                console.log('ERROR', error);
            }
        };
        renderStatusScreenBrick(bricksBuilder);
    </script>

</body>
</html>`

I get this error only in Safari, in chrome or firefox works perfect
**[Log] ERROR – SyntaxError: Invalid regular expression: invalid group specifier name (pepe.html, line 41)
SyntaxError: Invalid regular expression: invalid group specifier name**
Screenshot in safari
![Captura de pantalla 2022-12-25 a la(s) 23 40 39](https://user-images.githubusercontent.com/120674571/209492787-70ba3d69-5847-4f8d-b885-c8d5411d7aea.png)

Screenshot in chrome
![Captura de pantalla 2022-12-25 a la(s) 23 40 33](https://user-images.githubusercontent.com/120674571/209492814-10b025b6-77b3-4b26-855a-77d5d9078615.png)

**Reply by icaldana** [reply]
Hello @marianolop, how are you doing?
I'm sorry this is happening 😞 

Which is the Safari version? And iPhone's version - if you know?

**Reply by icaldana** [reply]
Hello @marianolop @dvmarthur 

A few notes:
- I've tested your script working fine on Safari 13+ and on Iphone 7+, thus, it would be nice to know the version you are using
- The Google script that wasn’t loading in your screenshot is on our radar already, but it should not break the page;

![image](https://user-images.githubusercontent.com/30550706/209663064-0fad53a2-e9c7-455e-9d31-7e5a536da1d7.png)

**Reply by marianolop** [reply]
Hi @icaldana Safari version is 16.6 in my Mac, and Iphone 13 real one and emulator.
One important thing if you try with a non existing or wrong payment, it will work. You have to do it with a existing payment. I tried the same as you, and with error the page works, but If you try with a true payment you will have the error I showed earlier.

![Captura de pantalla 2022-12-27 a la(s) 11 28 08](https://user-images.githubusercontent.com/120674571/209680713-ee59bbc4-9f21-4cfe-9bb2-303a302b1c77.png)

**Reply by marianolop** [reply]
@icaldana Please try to make a payment and then try the page with a succesfull payment and you'll it's not working

**Reply by icaldana** [reply]
Hello @marianolop,

I am sorry for the late response, I was able to reproduce the issue with a valid paymentId, and I am currently investigating the reason.  We will let you know once it is fixed.

Thank you!

**Reply by marianolop** [reply]
Thanks!!

**Reply by icaldana** [reply]
@marianolop Hello, how are you?
I'd like to let you know the fix is under development, and it will be deployed soon. I will let you know once it is available.
Thank you.

**Reply by marianolop** [reply]
Thanks!!!

**Reply by icaldana** [reply]
@marianolop hello! The fix is available! We appreciate your patience.
Thank you!

**Reply by marianolop** [reply]
Thanks!!! I'll test it!
